tests/rustdoc-json/primitives/no_primitive_impl_pollution.rs

```diff
@@ -0,0 +1,2 @@
+//@ !has "$.index[*][?(@.inner=='impl')]"
```

Comment:
I'm not sure if I've written this correctly, will this check for the presence of an object key, or just a string literal "impl"?